visualization/mlchartlib/src/PlotlyThemes.ts

```diff
@@ -36,47 +38,45 @@ const plotlyBlackTheme: IPlotlyTheme = {
 };
 
 export class PlotlyThemes {
-    public static applyTheme(props: IPlotlyProperty, theme?: string, themeOverride?: Partial<IPlotlyTheme>): IPlotlyProperty {
+    public static applyTheme(props: IPlotlyProperty, theme?: string | ITheme, themeOverride?: Partial<IPlotlyTheme>): IPlotlyProperty {
```

Comment:
@mizeljko The accesible charts should now be able to take in a Fabric theme object and appropriately be styled to match, while also keeping your change to allow overriding this when needed.